src/Detector/Hugo/HugoDetector.cs

```diff
@@ -100,6 +102,7 @@ private bool IsHugoApp(ISourceRepo sourceRepo)
                 {
                     if (IsHugoTomlFile(sourceRepo, tomlFile))
                     {
+                        directory = RelativeDirectoryHelper.GetRelativeDirectoryToRoot(tomlFile, sourceRepo.RootPath);
```

Comment:
What would be the app directory in this case? It should still be the parent directory of the `config` directory.

Reply:
I see, made changes to it